IAPCheck.ts

```diff
@@ -0,0 +1,38 @@
+
+enum TIMEOFYEAR {
+    FALL, 
+    SPRING
+}
+
+function statusToString(iapStatus: IAP_STATUS): string {
```

Comment:
```suggestion
function iapStatusToString(iapStatus: IAP_STATUS): string {
```

See the comment I made on the enum IAP_STATUS